src/app/shared/palette/date/write-date-field.component.ts

```diff
@@ -14,4 +14,8 @@ export class WriteDateFieldComponent extends AbstractFieldWriteComponent impleme
     this.dateControl = this.registerControl(new FormControl(this.caseField.value));
   }
 
+  isDateTime(): boolean {
+    return this.caseField.field_type.id === 'DateTime';
```

Comment:
field_type has got an 'id' and a 'type' property. I've seen sometime we use 'id' but mainly we use 'type' to check the type
to me feels more natural .type

Reply:
id and type differ for some types like AddressUK, AddressGlobal & AddressGlobalUK. For these types id is AddressUK while the type is Complex. For this one both mean the same...